src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -403,11 +403,18 @@ private Collection<FunctionDescriptor> ReadFunctions(ScriptHostConfiguration con
                         continue;
                     }
 
+                    if (functions.Count == ScriptConfig.MaximumFunctionCount)
+                    {
+                        AddFunctionError(functionName, string.Format(CultureInfo.InvariantCulture,
+                            "Maximun number of functions ({0}) exceeded. Ignoring function.", ScriptConfig.MaximumFunctionCount));
+                        continue;
```

Comment:
I considered a `break` but wanted to log each ignored function, since currently, we're not explicitly applying any kind of ordering and it would probably be helpful to know which functions are not running. I guess we also log which functions **are** running, so that may not be as important. I actually have a check in the test to make sure that is being logged as expected.
I'm don't have a strong opinion on this, so if you guys think the logging is irrelevant, I can change this to break out of the loop.